.github/workflows/pinot_tests.yml

```diff
@@ -221,18 +231,24 @@ jobs:
     if: github.repository == 'apache/pinot'
     runs-on: ubuntu-latest
     strategy:
+      # Changed to false in order to improve coverage using unsafe buffers
+      fail-fast: false
       matrix:
         # We only test LTS Java versions
         # TODO: add JDK 17 once ready. see: https://github.com/apache/pinot/issues/8529
-        java: [ 8, 11, 15 ]
+        java: [ 11, 17, 20 ]
+        distribution: [ "temurin" ]
+        include:
+          - java: 8
+            distribution: "adopt"
```

Comment:
This basically adds a new execution without adding it to the actual matrix. In this case, Java 8 is not included in `temurin`, so we need to bring it from `adopt`